keyboards/handwired/3dp660_oled/readme.md

```diff
@@ -0,0 +1,27 @@
+# handwired/3dp660_oled
+
+![handwired/3dp660_oled](https://i.imgur.com/AQ6VQfgh.jpg)
+
+*A handwired FC660 layout with OLED display support.*
+
+* Keyboard Maintainer: [xia0](https://github.com/xia0)
+* Hardware Supported: ATmega32U4 e.g. Arduino Pro Micro or Teensy 2.0
```

Comment:
pin matrix in `info.json` uses elite-c pins not used in a pro micro. Could you confirm this board requires an elite c microcontroller?

Reply:
d5 and b0 pins accessed from LED resistors as detailed here: https://golem.hu/guide/pro-micro-upgrade/

Reply:
Thanks for confirming, could you make note of this in `readme.md`?

Reply:
amended the readme. i've got it to work using atmel-dfu on hardware so i suppose it works?